ui/pages/confirmations/token-allowance/token-allowance.js

```diff
@@ -222,6 +237,43 @@ export default function TokenAllowance({
     });
   };
 
+  ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
+  const mmiApprovalFlow = () => {
+    if (accountType === 'custody') {
```

Comment:
Nit: Consider using an enum for `accountType` values to centralize and standardize.
Same for `custodyStatus = 'created'`.